README.md

```diff
@@ -132,6 +132,139 @@ PrestoGateway records history of recent queries and displays links to check quer
+To add a single resource group, specify all relevant fields in the body.
+```$xslt
+curl -X POST http://localhost:8080/presto/resourcegroup/create \
+ -d '{  "resourceGroupId": 1, \
+        "name": "resourcegroup1", \
+        "softMemoryLimit": "100%", \
+        "maxQueued": 100, \
+        "softConcurrencyLimit": 100, \
+        "hardConcurrencyLimit": 100, \
+        "environment": "test", \
+        "schedulingPolicy": null, \
+        "schedulingWeight": null, \
+        "jmxExport": null, \
+        "softCpuLimit": null, \
+        "hardCpuLimit": null, \
+        "parent": null, \
+        "environment": "test" \
+    }'
+```
+
+### Get existing resource group(s)
+If no resourceGroupId (type long) is specified, then all existing resource groups are fetched. 
+```$xslt
+curl -X GET http://localhost:8080/presto/resourcegroup/read?resourceGroupId={INSERT_ID_HERE}
+```
+
+### Update a resource group
+Specify all fields in the body, which will overwrite the existing resource group's properties.
+```$xslt
+curl -X POST http://localhost:8080/presto/resourcegroup/update \
+ -d '{  "resourceGroupId": 1, \
+        "name": "resourcegroup_updated", \
+        "softMemoryLimit": "80%", \
+        "maxQueued": 50, \
+        "softConcurrencyLimit": 40, \
+        "hardConcurrencyLimit": 60, \
+        "environment": "test", \
+        "schedulingPolicy": null, \
+        "schedulingWeight": null, \
+        "jmxExport": null, \
+        "softCpuLimit": null, \
+        "hardCpuLimit": null, \
+        "parent": null, \
+        "environment": "test" \
+    }'
+```
+
+### Delete a resource group
+To delete a resource group, specify the corresponding resourceGroupId (type long) in the body.
+```$xslt
+curl -X POST http://localhost:8080/presto/resourcegroup/delete \
+ -d '{  
+       {INSERT_ID_HERE}
+     }'
+```
+
+### Add a selector
+To add a single selector, specify all relevant fields in the body.
+```$xslt
+curl -X POST http://localhost:8080/presto/selector/create \
+ -d '{  "resourceGroupId": 1, \
+        "priority": 1, \
+        "userRegex": "selector1", \
+        "sourceRegex": "resourcegroup1", \
+        "queryType": "insert" \
+     }'
+```
+
+### Get existing selectors(s)
+If no resourceGroupId (type long) is specified, then all existing selectors are fetched. 
+```$xslt
+curl -X GET http://localhost:8080/presto/selector/read?resourceGroupId={INSERT_ID_HERE}
+```
+
+### Update a selector
+Specify all fields in the body, which will overwrite the existing selector's properties.
+```$xslt
+curl -X POST http://localhost:8080/presto/selector/update \
+ -d '{  "resourceGroupId": 1, \
+        "priority": 2, \
+        "userRegex": "selector_updated", \
+        "sourceRegex": "resourcegroup1", \
+        "queryType": "insert" \
+     }'
+```
+
+### Delete a selector
+To delete a selector, specify the corresponding resourceGroupId (type long) in the body.
+```$xslt
+curl -X POST http://localhost:8080/presto/selector/delete \
```

Comment:
can't this be `/delete?resourceGroupId={INSERT_ID_HERE}` for consistency?

Reply:
Yup, sounds good